test-files/cmd/glacial/NonDestructiveConflationTest.sh

```diff
@@ -17,6 +17,14 @@ hoot changeset-derive --stats \
  test-output/NonDestructiveTest/output.osm \
  test-output/NonDestructiveTest/changeset.osc
 
+# Check the output against the expected output
+hoot map-diff \
+  test-output/NonDestructiveTest/output.osm \
+  test-files/NonDestructiveTest/Expected.osm || \
+  diff \
+  test-output/NonDestructiveTest/output.osm \
+  test-files/NonDestructiveTest/Expected.osm
+
```

Comment:
Added a quick test to verify the output file is the same along with the stats below.  Also added the output file above.